### PR TITLE
Feat/add google pixel

### PR DESCRIPTION
**Pull request by olavoparno**
closes  https://github.com/Stand-With-Crypto/swc-internal/issues/223

fixes 

## How has it been tested?

- [X] Locally
- [x] Vercel Preview Branch
- [ ] Unit test
- [ ] Functional test

### CODE REVIEW

**Review by twistershark — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -160,4 +160,7 @@ BUILDER_IO_PUBLIC_KEY='123'
 BUILDER_IO_PRIVATE_KEY='123'
 
 # Github repository read access token
-GITHUB_TOKEN=''
\ No newline at end of file
+GITHUB_TOKEN=''
+
+# Google Tracking Manager
```
Comment (LucasRMP) [inline .env.example:165]: First thing, it's google tag manager, not tracking.
Second, let's add instructions on how to generate this for OS contributors

File: src/app/[locale]/layout.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { TopLevelClientLogic } from '@/app/[locale]/topLevelClientLogic'
 import { CookieConsent } from '@/components/app/cookieConsent'
 import { Footer } from '@/components/app/footer'
 import { GeoLocationDisclaimerBanner } from '@/components/app/geoLocationDisclaimerBanner/geoLocationDisclaimerBanner'
+import { GoogleTrackingManager } from '@/components/app/googleTrackingManager'
```
Comment (LucasRMP) [inline src/app/[locale]/layout.tsx:11]: Let's change this to `GoogleTagManager`, this is not the service name

File: src/components/app/googleTrackingManager.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { GoogleTagManager } from '@next/third-parties/google'
+
+import { requiredEnv } from '@/utils/shared/requiredEnv'
+import { NEXT_PUBLIC_ENVIRONMENT } from '@/utils/shared/sharedEnv'
+
+// even though locally we don't render GTM we still
+// want to capture on sentry if ID not set
+const googleTrackingManagerId = requiredEnv(
```
Comment (LucasRMP) [inline src/components/app/googleTrackingManager.tsx:8]:
```suggestion
const googleTrackingManagerId = requiredOutsideLocalEnv(
```

**Review by LucasRMP — CHANGES_REQUESTED** [review]

**Review by lucasrmp-cb — CHANGES_REQUESTED** [review]
Wrong account, comments above

**Review by eduardopicolo-cb — CHANGES_REQUESTED** [review]
`&loading=async` might be causing a bug in the politicians page. The google maps script is not loading after a page refresh. 
https://github.com/user-attachments/assets/c8ccc30b-bba6-4518-b855-1f0e558738bb

**Comment by olavoparno** [comment]
@eduardopicolo-cb thank you. Removed.

**Review by lucasrmp-cb — APPROVED** [review]

**Pull request merged**